cvat/apps/engine/media_extractors.py

```diff
@@ -179,25 +194,48 @@ def _make_name():
 
 class ZipReader(ImageListReader):
     def __init__(self, source_path, step=1, start=0, stop=None):
-        self._zip_source = zipfile.ZipFile(source_path[0], mode='r')
+        self._zip_source = zipfile.ZipFile(source_path[0], mode='a')
```

Comment:
What is a reason to change mode here?

Reply:
We add the converted (bin to PCD), into the in memory zip again, thus we require it to be "a" mode, against creating an entirely new zip